app/src/components/market/common_sections/card_top_details/market_top_details_closed/index.tsx

```diff
@@ -3,25 +3,44 @@ import React, { useState } from 'react'
 import { useHistory } from 'react-router'
 import styled from 'styled-components'
 
-import { useConnectedWeb3Context } from '../../../../contexts'
-import { useGraphMarketsFromQuestion } from '../../../../hooks/graph/useGraphMarketsFromQuestion'
-import { useTheme } from '../../../../hooks/useTheme'
-import { useWindowDimensions } from '../../../../hooks/useWindowDimensions'
-import { getContractAddress, getNativeAsset, getWrapToken } from '../../../../util/networks'
+import { useConnectedWeb3Context } from '../../../../../contexts'
+import { useGraphMarketsFromQuestion } from '../../../../../hooks/graph/useGraphMarketsFromQuestion'
+import { useTheme } from '../../../../../hooks/useTheme'
+import { useWindowDimensions } from '../../../../../hooks/useWindowDimensions'
+import { getContractAddress, getNativeAsset, getWrapToken } from '../../../../../util/networks'
 import {
   bigNumberToNumber,
   formatToShortNumber,
   getInitialCollateral,
   getMarketRelatedQuestionFilter,
   onChangeMarketCurrency,
-} from '../../../../util/tools'
-import { MarketMakerData, MarketState, Token } from '../../../../util/types'
-import { SubsectionTitleWrapper } from '../../../common'
+} from '../../../../../util/tools'
+import { MarketMakerData, MarketState, Token } from '../../../../../util/types'
+import { CurrencySelector } from '../../user_transactions_tokens/currency_selector'
 import { AdditionalMarketData } from '../additional_market_data'
-import { CurrencySelector } from '../currency_selector'
 import { MarketData } from '../market_data'
 import { ProgressBar } from '../progress_bar'
 import { ProgressBarToggle } from '../progress_bar/toggle'
+import { SubsectionTitleWrapper } from '../subsection_title_wrapper'
+
+// import { useConnectedWeb3Context } from '../../../../contexts'
+// import { useContracts } from '../../../../hooks'
+// import { useGraphMarketsFromQuestion } from '../../../../hooks/graph/useGraphMarketsFromQuestion'
+// import { useTheme } from '../../../../hooks/useTheme'
+// import { useWindowDimensions } from '../../../../hooks/useWindowDimensions'
+// import { getContractAddress, getNativeAsset, getWrapToken } from '../../../../util/networks'
+// import {
+//   bigNumberToString,
+//   getInitialCollateral,
+//   getMarketRelatedQuestionFilter,
+//   onChangeMarketCurrency,
+// } from '../../../../util/tools'
+// import { MarketMakerData, MarketState, Token } from '../../../../util/types'
+// import { SubsectionTitleWrapper } from '../../../common'
+// import { AdditionalMarketData } from '../additional_market_data'
+// import { MarketData } from '../market_data'
+// import { ProgressBar } from '../progress_bar'
+// import { ProgressBarToggle } from '../progress_bar/toggle'
```

Comment:
what is this?

Reply:
Looks like just got left behind from the merge... I've removed these lines